Foundation/NSNumberFormatter.swift

```diff
@@ -180,9 +179,17 @@ open class NumberFormatter : Formatter {
         
         set {
             switch newValue {
-            case .none, .ordinal, .spellOut:
+            case .none, .ordinal:
                 _usesSignificantDigits = false
-                
+
+            case .spellOut:
+                // FIXME: Locale identifier necessary to make unum work; cheat by using a "default"
+                // Should determine proper current locale identifier
+                if locale.identifier.length == 0 {
+                    _locale = Locale(identifier: "en_US")
```

Comment:
I am not certain regenerating the locale here is correct; it seems that the re-generation phase should have a locale based upon either the requested or the current locale.